easybuild/easyconfigs/f/FragGeneScan/FragGeneScan-1.31-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'FragGeneScan'
+version = '1.31'
+
+homepage = 'https://omics.informatics.indiana.edu/FragGeneScan/'
+description = "FragGeneScan is an application for finding (fragmented) genes in short reads."
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
```

Comment:
In case software is (might) be affected by the compiler performance, it's nicer to keep them at the GCC and iccifort level so that users can pick. So, even if we can move it to GCCcore, doesn't always mean we should.

If this is just a small utility, or when we know that performance isn't significantly better in iccifort, then GCCcore is the best choice. Is this the case here?

Reply:
The only tests I could do were example runs with included data. Measuring the execution time while having this compiled under GCC vs GCCcore didn't make any measurable difference. Thus leaving to GCCcore.

Reply:
There would never be any difference between GCC and GCCcore, the question is whether or not gcc and icc shows any performance differences. If icc is better, we don't want a version at GCCcore. 

I had a look at compiling both, and, either way, it seems like a small utility, so it might not be worth the hassle, and icc didn't perform any better anyway.